### PR TITLE
82 ys landing page nav

### PR DESCRIPTION
**Pull request by YoungSong99**
Screenshot 2024-10-15 at 2 35 51 PM

- [ ] Add nav for landing page
- [ ] Flash (notification, alert) disappear after 2 seconds

### CODE REVIEW

File: app/views/shared/_flash_notice.html.erb
Comment (MayaS1111) [inline app/views/shared/_flash_notice.html.erb:1]: This is a smart idea, thanks

File: app/views/shared/_landing_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div id="landing_nav">
+  <div class="flex flex-row justify-between pt-10 px-28 w-full">
+    <a href="/">
+      <img src="<%= asset_path("icon/Logo.svg") %>" alt="DPI Logo" class="h-[2.5rem]">
+    </a>
+    <div class="bg-white opacity-70 flex items-center	justify-around border-slate-300 rounded-2xl w-[26rem] h-[3.2rem]">
+      <ul class="flex gap-10 text-s font-normal text-black max-w-[26]">
+        <li><a href="/" class="hover:text-gray-600 font-semibold">Home</a></li>
+        <li><a href="/" class="hover:text-gray-600">Demo</a></li>
+        <li><a href="/" class="hover:text-gray-600">Dashboard</a></li>
```
Comment (MayaS1111) [inline app/views/shared/_landing_nav.html.erb:10]: Are you able to put the correct links?

File: app/views/shared/_landing_nav.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+             py-2 px-5 mr-auto hover:bg-transparent hover:text-[#202020] hover:border-[#202020]">
+          <%= button_to "Sign out", destroy_user_session_path, method: :delete, data: { turbo: false } %>
+        </div>
+      </div>
+    <% else %>
+      <div class="flex gap-4 items-center">
+        <div class="text-s text-[#676666]">
+          <%= link_to "Sign Up", new_user_registration_path %>
+        </div>
+        <%= link_to "Sign In", new_user_session_path, class: "text-s font-medium bg-[#202020] text-[#FFFFFF] border border-transparent rounded-md
```
Comment (MayaS1111) [inline app/views/shared/_landing_nav.html.erb:28]: What does the nave look like when a user is signed in?

**Review by MayaS1111 — CHANGES_REQUESTED** [review]

File: app/views/shared/_landing_nav.html.erb (hunk 2 above)
Comment (YoungSong99) [reply app/views/shared/_landing_nav.html.erb:28]: If user signed in they see Edit and Sign Out

File: app/views/shared/_landing_nav.html.erb (hunk 1 above)
Comment (YoungSong99) [reply app/views/shared/_landing_nav.html.erb:10]: I'll update the Dashboard link but Demo page is just an idea.
We can decide what to put for later

**Review by rezahusain — APPROVED** [review]
Looks great! Approved :)

**Review by APatel-AI — APPROVED** [review]
Looks good!

**Review by MayaS1111 — APPROVED** [review]
Awesome work Young!

**Pull request merged**